src/test/java/com/chikli/hudson/plugin/naginator/NaginatorListenerTest.java

```diff
@@ -45,46 +42,6 @@ protected void tearDown() throws Exception {
         }
     }
     
-    private final static class MyBuilder extends Builder {
-        private final String text;
-        private final Result result;
-        private final int duration;
-
-        public MyBuilder(String text, Result result) {
-            super();
-            this.text = text;
-            this.result = result;
-            this.duration = 0;
-        }
-
-        private MyBuilder(String text, Result result, int duration) {
-            this.text = text;
-            this.result = result;
-            this.duration = duration;
-        }
-
-        @Override
-        public boolean perform(AbstractBuild<?, ?> build, Launcher launcher,
-                               BuildListener listener) throws InterruptedException,
-                                                              IOException {
-            if (duration > 0) Thread.sleep(duration);
-
-            listener.getLogger().println(text);
-            build.setResult(result);
-            return true;
-        }
-
-        @Extension
-        public static final class DescriptorImpl extends Descriptor<Builder> {
-            public String getDisplayName() {
-                return "MyBuilder";
-            }
-            public MyBuilder newInstance(StaplerRequest req, JSONObject data) {
-                return new MyBuilder("foo", Result.SUCCESS);
-            }
-        }
-    }
-
```

Comment:
All changes in this file is to extract `MyBuilder` in order to make it usable from `NaginatorPublisherTest`